Failed: PrimitiveStateSpec

https://jenkins.akka.io:8498/job/akka-nightly/6069/testReport/junit/akka.persistence.typed.scaladsl/PrimitiveStateSpec/A_typed_persistent_actor_with_primitive_state_must_persist_events_and_update_state/

```
Error Message

java.lang.AssertionError: assertion failed: expected eventHandler:1:2, found onRecoveryCompleted:1

Stacktrace

sbt.ForkMain$ForkError: java.lang.AssertionError: assertion failed: expected eventHandler:1:2, found onRecoveryCompleted:1
	at scala.Predef$.assert(Predef.scala:219)
	at akka.actor.testkit.typed.internal.TestProbeImpl.expectMessage_internal(TestProbeImpl.scala:138)
	at akka.actor.testkit.typed.internal.TestProbeImpl.expectMessage(TestProbeImpl.scala:121)
	at akka.persistence.typed.scaladsl.PrimitiveStateSpec.$anonfun$new$2(PrimitiveStateSpec.scala:60)
	at akka.persistence.typed.scaladsl.PrimitiveStateSpec$$Lambda$967/1239911049.apply(Unknown Source)
```

--- a/akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PrimitiveStateSpec.scala
+++ b/akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PrimitiveStateSpec.scala
@@ -57,7 +57,7 @@ class PrimitiveStateSpec extends ScalaTestWithActorTestKit(PrimitiveStateSpec.co
       probe.expectTerminated(ref1)
 
       val ref2 = testKit.spawn(b)
-      // eventHandler from reply
+      // eventHandler from replay
       probe.expectMessage("eventHandler:0:1")
       probe.expectMessage("eventHandler:1:2")
       probe.expectMessage("onRecoveryCompleted:3")

--- a/akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PrimitiveStateSpec.scala
+++ b/akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PrimitiveStateSpec.scala
@@ -54,6 +54,8 @@ class PrimitiveStateSpec extends ScalaTestWithActorTestKit(PrimitiveStateSpec.co
       probe.expectMessage("eventHandler:1:2")
 
       ref1 ! -1
+      probe.expectTerminated(ref1)
+
       val ref2 = testKit.spawn(b)
       // eventHandler from reply
       probe.expectMessage("eventHandler:0:1")

